HasAttachmentTrait.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
```

Comment:
Why? I see that all other files has opening tags on the first line. Or do you mean to make comment block to match all other files?